nntrainer/app_context.cpp

```diff
@@ -47,8 +51,104 @@
 #include <nnstreamer_layer.h>
 #endif
 
+/// add #ifdef across platform
+static std::string layerlib_suffix = "layer.so";
+static const std::string func_tag = "[AppContext] ";
+
+constexpr const char *DEFAULT_CONF_PATH = "/etc/nntrainer.ini";
+
+constexpr const char *getConfPath() {
+#ifdef NNTRAINER_CONF_PATH
```

Comment:
How about set NNTRAINER_CONF_PATH to DEFAULT_CONF_PATH with `#define`. 
and let this function can simply return NNTRAINER_CONF_PATH.

Reply:
Done